webpack.config.js

```diff
@@ -165,10 +158,10 @@ const webpackConfig = {
 	},
 	plugins: [
 		new ForkTsCheckerWebpackPlugin(),
-		new FixStyleOnlyEntriesPlugin(),
+		new RemoveEmptyScriptsPlugin(),
```

Comment:
This does what `FixStyleOnlyEntriesPlugin` did, ensuring that any empty files/directories are removed.